frontend/components/buttons/Button/Button.tsx

```diff
@@ -12,10 +12,9 @@ export type ButtonVariant =
   | "text-link" // Underlines on hover
   | "text-icon"
   | "icon" // Buttons without text
-  | "small-icon" // Buttons without text
   | "inverse"
   | "inverse-alert"
-  | "unstyled"
+  | "unstyled" // Avoid as much as possible (used in registration breadcrumbs, 404/500, an old button dropdown)
```

Comment:
Was able to remove over half the instances of "unstyled", these last ones are tricky and not prioritized for this timebox ticket